packages/atomic/src/components/insight/user-actions/atomic-insight-user-action/atomic-insight-user-action.tsx

```diff
@@ -0,0 +1,97 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {UserAction} from '../..';
+import BookmarkIcon from '../../../../images/bookmark.svg';
+import DocumentIcon from '../../../../images/document.svg';
+import PointIcon from '../../../../images/point.svg';
+import QuickviewIcon from '../../../../images/quickview.svg';
+import SearchIcon from '../../../../images/search.svg';
+import {parseTimestampToDateDetails} from '../../../../utils/date-utils';
+import {InsightBindings} from '../../atomic-insight-interface/atomic-insight-interface';
+
+const icons = {
+  SEARCH: SearchIcon,
+  CLICK: DocumentIcon,
+  VIEW: QuickviewIcon,
+  CUSTOM: PointIcon,
+  TICKET_CREATION: BookmarkIcon,
+};
+
+interface AtomicInsightUserAction {
+  action: UserAction;
+  bindings: InsightBindings;
+}
+
+export const AtomicInsightUserAction: FunctionalComponent<
+  AtomicInsightUserAction
+> = ({bindings, action}) => {
```

Comment:
More of a nitpick, feel free to keep it like it is now. 

If this is simply a functional component only used in `atomic-insight-user-actions-session`, I would not prefix it with 'Atomic' and I would put it in the same folder as `atomic-insight-user-actions-session`. I think I would just call it `<Action/>`. Having it prefixed with 'Atomic' in its separate folder brings confusion as we all first think its an exported component.

Reply:
good idea! refactor applied.